scipy/optimize/rectangular_lsap/rectangular_lsap.cpp

```diff
@@ -42,6 +42,7 @@ Author: PM Larsen
 
 #include <algorithm>
 #include <cmath>
+#include <cstdint>
```

Comment:
Looks like `<cstdint>` is failing the build as it's C++11 only. You could change the config to build with C++11 but it's probably easier to just change the include. 
```suggestion
#include "rectangular_lsap.h"
```

Reply:
Good point, thanks! :)